email/README.md

```diff
@@ -214,3 +214,90 @@ email:
 
 When an email is sent via the `email-test` service above, it will not actually
 be sent. The email contents will be printed on the console.
+
+## Sending an Email to the user an OTP to log in
+
+One of the most common use cases of using an Email Service is to enable 2FA authentication. Here is a small example of how you can setup share OTP using Email with Gramex.
+
+Here are the steps that need to be followed to execute this exercise successfully:-
+
+- Create an event using an object which triggers the event of sending the OTP. This is usually triggered on a click of a button but can be based on any other object.
```

Comment:
- [ ] The language here suddenly becomes technical. Can we make it simpler, and shorter? At a cursory read, I was able to understand the first para completely. I couldn't do that here
- [ ] Let's make any steps an ordered list